Build/libHttpClient.Android/build.gradle

```diff
@@ -1,11 +1,11 @@
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion 27
-    ndkVersion "22.0.7026061"
+    compileSdkVersion 30
+    ndkVersion '22.0.7026061'
 
     defaultConfig {
-        targetSdkVersion 21
+        targetSdkVersion 30
```

Comment:
don't change the target sdk, this is what defines our minimum supported version

Reply:
@lubetra, I think you're thinking of the `minSdkVersion`. `targetSdkVersion` defines what SDK level you're intending to run against, which enables/disables various back-compat and security features.